src/commands/workflow/run.ts

```diff
@@ -0,0 +1,48 @@
+import { Flags } from "@oclif/core";
+
+import * as ApiV1 from "@/lib/api-v1";
+import BaseCommand from "@/lib/base-command";
+import { commaSeparatedStr, jsonStr } from "@/lib/helpers/flag";
+import { withSpinner } from "@/lib/helpers/request";
+import { indentString } from "@/lib/helpers/string";
+
+export default class WorkflowRun extends BaseCommand {
+  static flags = {
+    environment: Flags.string({
+      default: "development",
+      summary:
+        "The environment in which to run the workflow; defaults to development.",
```

Comment:
```suggestion
        "The environment in which to run the workflow",
```
Oclif displays the default value in the --help doc automatically.